cluster/saltbase/salt/calico/10-calico.conf

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "calico-k8s-network",
+    "type": "calico",
+    "etcd_authority": "{{ grains.api_servers }}:6666",
```

Comment:
The plurality of `grains.api_servers` has me a bit worried.  I _think_ this is just a single instance, but if this ever switches to multi-master then this could all break when there are multiple apiserver instances.